.swiftlint.yml

```diff
@@ -1,56 +1,62 @@
 # .swiftlint.yml
 #
-# Useful rules reference: https://realm.github.io/SwiftLint/rule-directory.html
+# Goals:
+#   - Decrease number of `disabled_rules` that we agree with as a team
+#   - Increase number of `opt_in_rules` that we agree with as a team
+#   - Increase number of `custom_rules` that we agree with as a team
+#   - Avoid rule customization unless increasing strictness that we agree with as a team
+#
+# Useful rules reference: https://github.com/realm/SwiftLint/blob/master/Rules.md
 
 disabled_rules:
+  - cyclomatic_complexity       # This finds some hairy methods
+  - file_length
+  - force_cast
+  - function_body_length        # Interesting overlap between these violations and cyclomatic_complexity
   - line_length
+  - operator_whitespace
+  - todo
   - type_body_length
+  - void_function_in_ternary
+  - force_try
+  - opening_brace
+  - closing_brace
+  - for_where
+  - function_parameter_count
+  - multiple_closures_with_trailing_closure
 
 opt_in_rules:
-  - attributes
-  - closing_brace
   - colon
   - comma
   - control_statement
+  - discouraged_object_literal
   - empty_count
-  - file_header
```

Comment:
disabled because there are multiple header patterns because of auto generated files, it is too hard to include all the patterns or mark exclusions.